src/App.tsx

```diff
@@ -1,19 +1,38 @@
 import { Card } from "./components";
 import "./app.css";
+import { useState } from "react";
 
 const App = () => {
+
   const url =
     "https://images.pexels.com/photos/1404819/pexels-photo-1404819.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2";
+  
+  const [position, setPosition] = useState('top');
+  
+  const setTop = () => {
+    setPosition('top');
+    }
 
-  return (
+  const setBottom = () => {
+    setPosition('bottom');
+    }
+     
+    return (
+      <>
     <main className="main-section">
       <Card
         title="I am Title"
         text="Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore, voluptatibus?"
         imgSrc={url}
-        position="bottom"
-      />
+        imgPosition={position}
+      /> 
+      
+      <div className="button">
+        <button onClick={() => setTop()}>Top</button>
+        <button onClick={() => setBottom()}>Bottom</button>
```

Comment:
I think it will be simpler if we do it like this. Then we do not need `setTop`and `setBottom` handlers !
```suggestion
        <button onClick={() => setPosition('top')}>Top</button>
        <button onClick={() => setPosition('bottom')}>Bottom</button>
```